models/src/version.rs

```diff
@@ -8,7 +8,7 @@ use std::{convert::TryInto, fmt, str::FromStr};
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 pub struct Version {
     instance: [char; 4], // ascii uppercase alpha
-    major: u32,
+    pub major: u32,
```

Comment:
Why not make all fields of this struct `pub`?

Reply:
Good question

Reply:
in what case do we need to directly access any of those fields ?

Reply:
I used major to know which testnet will end

Reply:
Makes me wonder if this PR is useful enough to justify making such things public... especially since this whole PR will be reverted at the end of the testnet, so the time spent working on it should be extremely limited

Reply:
What's the issue in making that public ? From what I understand after @yvan-sraka 's work on the config it will be available everywhere anyway

Reply:
the less public things we expose, the better. Here we're talking about version.major, which is not only used in config, but also in inter-node communication, and will be used for versioning in the future. Remember when we did a whole bunch of "pub" removals ?

Reply:
Done

Reply:
Removed the pub